PatientRegistrationApp/src/components/EligibilityQuestions.vue

```diff
@@ -0,0 +1,201 @@
+          <v-col cols="12" sm="12" md="6" lg="6"><v-checkbox v-model="eligibilityQ6" label="Down Syndrome" value="condition4"></v-checkbox></v-col>
+          <v-col cols="12" sm="12" md="6" lg="6"><v-checkbox v-model="eligibilityQ6" label="Heart conditions, such as heart failure, coronary artery disease, or cardiomyopathies" value="condition5"></v-checkbox></v-col>
+          <v-col cols="12" sm="12" md="6" lg="6"><v-checkbox v-model="eligibilityQ6" label="Immunocompromised state (weakened immune system) from solid organ transplant or from blood or bone marrow transplant, immune deficiencies, HIV, use of corticosteroids, or use of other immune weakening medicines" value="condition6"></v-checkbox></v-col>
+          <v-col cols="12" sm="12" md="6" lg="6"><v-checkbox v-model="eligibilityQ6" label="Obesity (body mass index [BMI] of 30 kg/m2 or higher but < 40 kg/m2)" value="condition7"></v-checkbox></v-col>
+          <v-col cols="12" sm="12" md="6" lg="6"><v-checkbox v-model="eligibilityQ6" label="Severe Obesity (BMI ≥ 40 kg/m2)" value="condition8"></v-checkbox></v-col>
+          <v-col cols="12" sm="12" md="6" lg="6"><v-checkbox v-model="eligibilityQ6" label="Pregnancy" value="condition9"></v-checkbox></v-col>
+          <v-col cols="12" sm="12" md="6" lg="6"><v-checkbox v-model="eligibilityQ6" label="Sickle cell disease" value="condition10"></v-checkbox></v-col>
+          <v-col cols="12" sm="12" md="6" lg="6"><v-checkbox v-model="eligibilityQ6" label="Smoking" value="condition11"></v-checkbox></v-col>
+          <v-col cols="12" sm="12" md="6" lg="6"><v-checkbox v-model="eligibilityQ6" label="Type 2 diabetes mellitus" value="condition12"></v-checkbox></v-col>
+          <v-col cols="12" sm="12" md="6" lg="6"><v-checkbox v-model="eligibilityQ6" label="None of the above" value="condition13"></v-checkbox></v-col>
+    </v-row>
+    <v-row no-gutters>
+      <v-divider></v-divider>
+    </v-row>
+  </v-container>
+</template>
+
+<script>
+import EventBus from '../eventBus'
+
+export default {
+  name: "eligibilityQuestions",
+  data() {
+    return {
+        county: '',
+		eligibilityQ2: '',
+		eligibilityQ3: '',
+		eligibilityQ4: '',
+		eligibilityQ5: '',
+		eligibilityQ6: [''],
+      valid: false,
+    };
+  },
+  computed: {
+      rules() {
+      return [
+        this.selected.length > 0 || "At least one item should be selected"
+      ];
+    }
+  },
+  methods: {
+    sendeligibilityResponsesToReviewPage() { //do we want to send it anywhere?
+      const eligibilityResponsesPayload = {
+        county: this.county,
+        eligibilityQ3: this.eligibilityQ2,
+        eligibilityQ4: this.eligibilityQ3,
+        eligibilityQ5: this.eligibilityQ4,
+        eligibilityQ6: this.eligibilityQ5,
+        eligibilityQ7: this.eligibilityQ6,
+      }
+      EventBus.$emit('DATA_eligibility_RESPONSES_PUBLISHED', eligibilityResponsesPayload)
+    },
+    verifyFormContents() {
```

Comment:
Maybe use v-form and it's validate method https://vuetifyjs.com/en/components/forms/#validation-with-submit-26-clear?

Reply:
There is a card in the backlog that we recently groomed that addresses changing the validation methods to v-form instead!